vscode-car-plugin/src/main/java/org/wso2/maven/MavenUtils.java

```diff
@@ -108,15 +108,15 @@ public void consumeLine(String line) {
      * @param project Maven project
      * @return whether to pack connector dependencies or not
      */
-    public static boolean ignoreConnectorDependencies(MavenProject project) {
+    public static boolean isConnectorPackingSupported(MavenProject project) {
         String runtimeVersion = project.getProperties().getProperty(Constants.PROJECT_RUNTIME_VERSION);
 
         if (runtimeVersion != null) {
             // Compare the version
             ComparableVersion currentVersion = new ComparableVersion(runtimeVersion);
             ComparableVersion targetVersion = new ComparableVersion(Constants.RUNTIME_VERSION_440);
 
-            return currentVersion.compareTo(targetVersion) < 0;
+            return currentVersion.compareTo(targetVersion) > 0;
```

Comment:
Doesn't it have to be >= ?

Reply:
Fixed in be48ba331b0579dcbf09b4cfc95daf4b513c4445